engine/env/cluster/src/test/java/org/datacleaner/cluster/DistributedAnalysisRunnerTest.java

```diff
@@ -118,7 +118,7 @@ public void testErrorHandlingFourSlaves() throws Exception {
 
     public void testUndistributableAnalyzer() throws Exception {
         final Datastore datastore = TestHelper.createSampleDatabaseDatastore("orderdb");
-        final AnalyzerBeansConfiguration configuration = new AnalyzerBeansConfigurationImpl()
+        final DataCleanerConfiguration configuration = new AnalyzerBeansConfigurationImpl()
```

Comment:
I guess it's one of many cases where the old stuff is still in use. I didn't get around to fix ALL the deprecation warnings.